From 48ca4c49de6d2519e023e99d1d18e7b731a73324 Mon Sep 17 00:00:00 2001
From: "Ye.Li" <B37916@freescale.com>
Date: Tue, 1 Sep 2015 17:25:15 +0800
Subject: [PATCH] MLK-11478 imx: mx6ul: Add QSPINOR boot support on mx6ul 9x9
 evk board

Add new build target: mx6ul_9x9_evk_qspi1_config

Signed-off-by: Ye.Li <B37916@freescale.com>
---
 configs/mx6ul_9x9_evk_qspi1_defconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configs/mx6ul_9x9_evk_qspi1_defconfig b/configs/mx6ul_9x9_evk_qspi1_defconfig
new file mode 100644
index 0000000..69c5f8a
--- /dev/null
+++ b/configs/mx6ul_9x9_evk_qspi1_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6ul_14x14_evk/imximage_lpddr2.cfg,MX6UL,MX6UL_9X9_LPDDR2,SYS_BOOT_QSPI"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6UL_14X14_EVK=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
-- 
1.8.0

